### PR TITLE
Don't alllow numbers as transform value type

### PR DESCRIPTION
**Pull request by wcandillon**

### CODE REVIEW

File: react-native-reanimated-tests.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* eslint-disable */
 import React, { useState } from 'react';
-import { StyleSheet, Button, View } from 'react-native';
+import { StyleSheet, Button, View, ViewStyle } from 'react-native';
```
Comment (Andreyco) [inline react-native-reanimated-tests.tsx:3]: `ViewStyle` is not used in this file.

Comment (wcandillon) [reply react-native-reanimated-tests.tsx]: thks, done

**Review by jakub-gonet — APPROVED** [review]

File: react-native-reanimated.d.ts [hunk 2]
```diff
@@ -197,13 +197,13 @@ declare module 'react-native-reanimated' {
             ? ReadonlyArray<AnimateStyle<S[K][0]>>
             : S[K] extends object
             ? AnimateStyle<S[K]> // allow `number` where `string` normally is to support colors
```
Comment (jakub-gonet) [inline react-native-reanimated.d.ts:199]: This comment is stale I believe.

**Comment by jakub-gonet** [comment]
@wcandillon 
I updated the branch and removed the stale commit.

**Pull request merged**